dbt/models/model/docs.md

```diff
@@ -188,4 +188,4 @@ View to compile PIN-level model inputs shared between the residential
 (`model.vw_card_res_input`) and condo (`model.vw_pin_condo_input`) model views.
 
 **Primary Key**: `year`, `run_id`, `meta_pin`
```

Comment:
**suggestion:** None of the `model.vw_` views actually have `run_id` as a component of their primary key. Would be nice to remove it for those views.